test/integration/forms/action.js

```diff
@@ -251,19 +289,29 @@ context('Patient Action Form', function() {
       },
     });
 
+    const testPatient = getPatient();
+
+    const testAction = getAction({
+      relationships: {
+        'form': getRelationship(testForm),
+        'form-responses': getRelationship([formResponse]),
+      },
+    });
+
+    localStorage.setItem(`form-subm-${ currentClinician.id }-${ testPatient.id }-${ testForm.id }-${ testAction.id }`, JSON.stringify({
+      updated: testTsSubtract(1),
+      submission: {
+        fields: { foo: 'foo' },
+      },
+    }));
+
     cy
       .routeAction(fx => {
-        fx.data = getAction({
-          id: '1',
-          relationships: {
-            'form': getRelationship('11111', 'forms'),
-            'form-responses': getRelationship([formResponse]),
-          },
-        });
+        fx.data = testAction;
 
         return fx;
       })
-      .routeFormByAction(_.identity, '11111')
+      .routeFormByAction(_.identity, testForm.id)
```

Comment:
I kinda think we could remove the 2nd argument.. it's a strange pattern now.. also we can probably default `routeFormByAction` to return the `testForm` and make everything else the exception

Reply:
actually it looks like you did remove it, so this isn't doing anything.. no need to pass the `_.identity`s

Reply:
Ya I missed the removal of a couple of these. They're gone now 👋